lib/index.js

```diff
@@ -40,20 +40,25 @@ if (typeof process.env.NODE_PG_FORCE_NATIVE !== 'undefined') {
   module.exports = new PG(Client)
 
   // lazy require native module...the native module may not have installed
-  module.exports.__defineGetter__('native', function () {
-    delete module.exports.native
```

Comment:
This implementation had to change somewhat since `Object.defineProperty` doesn't like deleting a getter. If the goal was to only run the getter once and then cache the result, I think I've reimplement that in a way that's more standard.